geopandas/io/tests/test_sql.py

```diff
@@ -75,6 +76,95 @@ def connection_spatialite():
     con.close()
 
 
+def connect_engine(dbname, user=None, password=None, host=None, port=None):
+    """
+    Initiaties a connection (engine) to a postGIS database that must already exist.
+    See create_postgis for more information.
+    """
+    sqlalchemy = pytest.importorskip("sqlalchemy")
+    from sqlalchemy.engine.url import URL
```

Comment:
Can you add a `pytest.importorskip("psycopg2")` here?

Reply:
Added this for engine creation fixture as well.